code/modules/research/xenoarchaeology/tools/anomaly_suit.dm

```diff
@@ -23,10 +23,16 @@
 	item_state = "cespace_suit"
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 100, rad = 100)
 	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank)
+	species_fit = null
+	sprite_sheets = null
+	sprite_sheets_obj = null
 
 /obj/item/clothing/head/helmet/space/eva/anomaly
 	name = "Excavation hood"
 	desc = "A pressure resistant excavation hood partially capable of insulating against exotic alien energies."
 	icon_state = "cespace_helmet"
 	item_state = "cespace_helmet"
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 100, rad = 100)
+	species_fit = null
+	sprite_sheets = null
+	sprite_sheets_obj = null
```

Comment:
There is no sprite for either the vulp or vox for these. So these are null for a reason.